<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="redButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(200,20,20);
    border: 3px solid black;
    width: 50px;
    height: 50px;
    border-radius: 25px;
    padding: 5px;
}
QPushButton:pressed {
    background-color: red;
}
QPushButton:hover {
    background-color: rgb(225,90,90);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="blueButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(20,20,200);
    border: 3px solid black;
    width: 50px;
    height: 50px;
    border-radius: 25px;
    padding: 5px;
}
QPushButton:pressed {
    background-color: blue;
}
QPushButton:hover {
    background-color: rgb(90,90,225);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#startButton {
    color: white;                      
    background-color: green;   
    font-size: 18px;                   
    font-weight: bold;                 
    border: none;                      
    border-radius: 10px;                
    padding: 10px 20px;                 
    text-align: center;                
}
QPushButton#startButton:hover {
    background-color: darkgreen; 
}</string>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="nightmareModeCheckBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Nightmare Mode</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>441</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 7px;
    background: white;
    text-align: center;      
    font-weight: bold;             
    color: black;                  
}
QProgressBar::chunk {
    border-radius: 5px; 
    background-color: cyan
}
</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="gameOverLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#gameOverLabel {
    color: red; 
    background-color: rgba(0, 0, 0, 70%); 
    font-size: 24px;               
    font-weight: bold;              
    border: 2px solid red;        
    border-radius: 10px;          
    padding: 10px;               
    text-align: center; 
	qproperty-alignment: 'AlignCenter';           
}</string>
    </property>
    <property name="text">
     <string>Game Over!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
